usaspending_api/data/testing_data/endpoint_testing_data.json

```diff
@@ -525,7 +525,7 @@
             ]
```

Comment:
I think this was mainly a stop-gap for our previous lack of testing on each endpoint. I'd love to migrate these out into their own `test_whatever_endpoint.py` files to follow normal testing conventions. As it is, I've been trying to add new all tests to their own test file, and just keep this one updated for any previous tests. (This way we can also reduce reliance on "endpoint_fixture_db" and use our model generator for test data)